.github/workflows/load-testing.yml

```diff
@@ -104,12 +109,32 @@ jobs:
           make kind-install-kyverno-from-repo
       - name: Wait for kyverno ready
         uses: ./.github/actions/kyverno-wait-ready
+      - name: Verify TLS secrets exist
```

Comment:
How does this changes relate to the actual topic of your PR? I would prefer to work only on one topic per PR